homeassistant/package_constraints.txt

```diff
@@ -25,7 +25,7 @@ ruamel.yaml==0.15.100
 sqlalchemy==1.3.17
 voluptuous-serialize==2.3.0
 voluptuous==0.11.7
-zeroconf==0.26.3
+zeroconf==0.27.1
```

Comment:
Since zeroconf is specified in the constraints file, we shouldn't have to update it everywhere. We install packages with `pip install --constraints homeassistant/package_constraints.txt REQUIREMENTS`, so that way we are always installing the version that `zeroconf` depends on. Obviously if users don't have `zeroconf` integration loaded this won't be installed.